CMakeLists.txt

```diff
@@ -544,14 +544,12 @@ IF (CXXTEST_FOUND)
 		)
 	ENDIF (CMAKE_BUILD_TYPE STREQUAL "Coverage")
 	ADD_CUSTOM_TARGET(test_python
-		DEPENDS PythonEvalUTest
-				PythonUtilitiesUTest
-				bindlink_cython atomspace_cython
-				scheme_wrapper logger_cython utilities_cython
+		DEPENDS PythonEval logger_cython scheme_wrapper bindlink_cython atomspace_cython
 		WORKING_DIRECTORY tests/cython
 		COMMAND ${CMAKE_CTEST_COMMAND} $(ARGS)
 		COMMENT "Running Python and Cython tests..."
 	)
+	add_dependencies(tests PythonEval logger_cython scheme_wrapper bindlink_cython atomspace_cython)
```

Comment:
Why is this needed? The tests themselves should already declare the correct dependencies; you should not need to also declare additional dependencies here.

Reply:
I didn't figure out how, `add_dependencies` won't work on tests, so I used this workaround.